app/src/main/java/org/thoughtcrime/securesms/ui/color/ColorDefs.kt

```diff
@@ -0,0 +1,53 @@
+package org.thoughtcrime.securesms.ui.color
+
+import androidx.compose.ui.graphics.Color
+
```

Comment:
The Colors.kt is a standard Compose file used to define Colors.
I did some clean up in my branch, a lot of it is now lost due to the many changes since.
But theme related content should be in Themes.kt, and Colors.kt should mostly be color definition

Reply:
It's all just preference, no?

I thought it made sense to get all of these color definitions in their own file, as I had a bunch of `Color`s and then the 4 `Colors` classes, so this file was getting bloated.

And `Colors` aren't themes, no? `Theme = colors + shapes + typogaphy`, no? or does your idea of theme just include colors? Would you have a custom `Theme` object?

Reply:
This was all started or done in the other branch, but yeah the Colors.kt is, traditionally, in a Compose project, where you define colors. Hence why I had moved a lot of what you had in that file in the Theme file.
The Theme file is about opinionated sets, for example a set of colors.
Again, as per the other comment, I'm happy to do this in a separate PR is all these things are holding things back. I just thought it was easy enough to integrate which is why I did it in a separate PR but things have changed a fair bit in parallel.

Reply:
cool